pkg/ddevapp/nodejs_test.go

```diff
@@ -41,7 +40,8 @@ func TestNodeJSVersions(t *testing.T) {
 	err = app.Start()
 	require.NoError(t, err)
 
-	for _, v := range nodeps.GetValidNodeVersions() {
+	// Testing some random versions
+	for _, v := range []string{"6", "10", "14", "16", "20"} {
```

Comment:
Shouldn't this have some examples with minor versions?

Reply:
Sure